cli/.js/src/main/scala/aqua/builder/GetterBuilder.scala

```diff
@@ -0,0 +1,41 @@
+package aqua.builder
+
+import aqua.js.{CallJsFunction, CallServiceHandler, FluencePeer}
+import aqua.model.func.Call
+import aqua.model.func.raw.CallServiceTag
+import aqua.model.{LiteralModel, VarModel}
+
+import scala.concurrent.Promise
+
+// Service that can return argument to use it from a code
+case class GetterBuilder(serviceId: String, value: VarModel, arg: scalajs.js.Dynamic) {
```

Comment:
It seems like a service, not a builder.
You make a service with this ID, then you register it. Nothing is built.